mysql-test/r/mysql_config_editor.result

```diff
@@ -100,14 +100,8 @@ user = test_user4
 ##############################################
 # Tests for mysql_config_editor's help command
 ##############################################
-<<<<<<< HEAD
 Copyright (c) 2009-2021 Percona LLC and/or its affiliates
```

Comment:
Shall we change year to 2009-2020 here?

Reply:
Ok, I see this is done in next commit. Would it be better to move this change to "test fixes" then?